kafka-admin/src/main/java/io/strimzi/admin/kafka/admin/handlers/TopicListHandler.java

```diff
@@ -0,0 +1,38 @@
+package io.strimzi.admin.kafka.admin.handlers;
+
+import graphql.schema.DataFetchingEnvironment;
+import io.strimzi.admin.kafka.admin.model.Topic;
+import io.vertx.core.Promise;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+public class TopicListHandler {
+    private static final List<Topic> sampleTopics = new ArrayList<>();
+    static {
+        sampleTopics.add(Topic.create("topic1", false, 3, 3));
+        sampleTopics.add(Topic.create("topic2-a", false, 1, 1));
+        sampleTopics.add(Topic.create("topic2-b", false, 1, 1));
+        sampleTopics.add(Topic.create("topic2-c", false, 1, 1));
+        sampleTopics.add(Topic.create("topic3", false, 6, 1));
+    }
```

Comment:
And reason to duplicate these, when they're the values of `TopicHandler#sampleTopics`?